dist/profile/templates/azure.ci.jenkins.io/tools.yaml.erb

```diff
@@ -56,6 +56,30 @@ tool:
               label: "s390x"
               subdir: "jdk-<%= @tools["jdk11"]["version"] %>"
               url: "<%= @tools["jdk11"]["sourceURL"] %>/jdk-<%= @tools["jdk11"]["version"] %>/OpenJDK11U-jdk_s390x_linux_hotspot_<%= @tools["jdk11"]["version"].gsub('+', '_') %>.tar.gz"
+    - name: "jdk17"
+      properties:
+      - installSource:
+          installers:
+          - zip:
+              label: "linux && amd64"
+              subdir: "jdk-<%= @tools["jdk17"]["version"] %>"
+              url: "<%= @tools["jdk17"]["sourceURL"] %>/jdk-<%= @tools["jdk17"]["version"] %>/OpenJDK17-jdk_x64_linux_hotspot_<%= @tools["jdk17"]["version"].gsub('+', '_') %>.tar.gz"
```

Comment:
I got the resulting URL on vagrant, which is 404 Not found: https://github.com/adoptium/temurin11-binaries/releases/download/jdk-17+35/OpenJDK17-jdk_x64_linux_hotspot_17_35.tar.gz

Reply:
hmm s/11/17 somewhere

Reply:
fixed